### DISCUSSION TITLE
Signal-cli for stretch (Debian 10.4) 32b

### DISCUSSION THREAD

**Discussion by Mike59FR** [AsamK/signal-cli#1118] [Q&A] [answered]
Hello,
I am under Stretch 32 bits (Raspberry Pi 3).
I installed JDK 17.0.1
I can't find the correct release of signal-cli and module (source or binary) of libsignal_jni.so that works. If you can help me? Thanks.

**Comment by orazioedoardo** [comment, marked as answer]
Debian 10.4 is buster, not stretch. The 32bit ARM version [has problems](https://github.com/exquo/signal-libs-build/issues/13) cross-compiling. I would reimage the flash card with the 64 bit version of bullseye and use the aarch64 version of libsignal_jni.so
